src/package.xml

```diff
@@ -1,104 +1,44 @@
 <?xml version="1.0" encoding="utf-8"?>
-<package name="helloworldTimePy" displayName="Example demonstrating how to use timesteps with a python model" version="1.0.0" url="https://apexrms.github.io/helloworldEnhanced/">
-	<transformers>
+<package name="helloworldTimePy" displayName="Hello World Time (Python)" description = "Example demonstrating how to use timesteps with a python model" version="2.0.0" url="https://apexrms.github.io/helloworldEnhanced/">
 
-		<!--Primary Transformer-->
-		<transformer
-			name="Primary"
-			isPrimary="True"
-			programName="python"
-			programArguments="model.py">
+	<!--Scenario Datasheets-->
 
-			<!--Chart Transformer-->
-			<include>
-				<transformer name="corestime_Runtime"/>
-			</include>
+	<!--Run Control-->
+	<dataSheet name="RunControl" displayName="Run Control" isRunControl="True" timestepTitle="Timestep" showMinIteration="False"/>
 
-			<datafeeds>
+	<!--Model Inputs-->
+	<dataSheet name="InputDatasheet" isSingleRow="True">
+		<column name="m" dataType="Double" displayName="Value for m" />
+		<column name="b" dataType="Integer" displayName="Value for b" />
+	</dataSheet>
 
-				<!--Run Control-->
-				<datafeed name="RunControl" displayName="Run Control" dataScope="Scenario">
-					<datasheets>
-						<datasheet name="RunControl" displayName="Run Control" isSingleRow="True">
-							<columns>
-								<column name="RunControlID" dataType="Integer" isPrimary="True"/>
-								<column name="ScenarioID" dataType="Integer"/>
-								<column
-									name="MinimumTimestep"
-									displayName="Minimum Timestep"
-									dataType="Integer"
-									defaultValue="0"
-									validationType="WholeNumber"
-									validationCondition="GreaterEqual"
-									formula1="0"/>
-								<column
-									name="MaximumTimestep"
-									displayName="Maximum Timestep"
-									dataType="Integer"
-									defaultValue="10"
-									validationType="WholeNumber"
-									validationCondition="GreaterEqual"
-									formula1="0"/>
-							</columns>
-						</datasheet>
-					</datasheets>
-				</datafeed>
+	<!--Model Output-->
+	<dataSheet name="OutputDatasheet">
+		<column name="Iteration" dataType="Integer" isOptional="True" />
```

Comment:
The reason why the charts aren't working is because iteration and timestep are included in the `dataSheet` tag instead of being defined as columns for the datasheet. So, if you replace what you have here for the output datasheet with the code below it should work. 

```
	<dataSheet name="OutputDatasheet" hasTimestep="True">
		<column name="y" dataType="Double" displayName="Value for y" />
	</dataSheet>
```